templates/terraform/update_mask.erb

```diff
@@ -1,5 +1,4 @@
 updateMask := []string{}
-
 <% update_body_properties.each do |prop| -%>
 <%# UpdateMask documentation is not not obvious about which fields are supported or
```

Comment:
Looks like this comment is out of date now

Reply:
Updated.